GiniCapture/Classes/Core/Screens/Document picker/Gallery/AlbumsPickerViewController.swift

```diff
@@ -82,10 +105,34 @@ extension AlbumsPickerViewController: UITableViewDataSource {
                     galleryManager: galleryManager)
         return cell!
     }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        if #available(iOS 14.0, *) {
+            let frame: CGRect = tableView.frame
+            let buttonTitle = NSLocalizedString("ginicapture.albums.selectMorePhotosButton",
```

Comment:
We have the `NSLocalizedStringPreferredFormat()` helper to allow clients to override the string. Did you intentionally not use it to not allow clients to customize the string?

Reply:
No, I've forgot about this method.